### PR TITLE
Master

### PR DESCRIPTION
**Pull request by Ouadie**
Merging the bugfix of @michaelcho and adding *bower.js*

### CODE REVIEW

**Comment by Ouadie** [comment]
any comment about this pull-request ? @yuku-t

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "jquery.overlay",
+  "version": "0.0.4",
+  "homepage": "https://github.com/yuku-t/jquery-overlay",
+  "authors": [
+    "Ouadie BOUSSAID <ouadie.boussaid@gmail.com>"
```
Comment (yuku-t) [inline bower.json]: lol

Comment (Ouadie) [inline bower.json]: yep, was generated by `bower init` you can accept the pull request then changed it..

**Comment by yuku-t** [comment]
Thanks for the patch.
I cleaned up commits log and fixed bower.json then merged the branch into master.

**Pull request closed without merging**